deploy/web_scripts/notify_build.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+
+# Accept a web-hook from GitHub telling us about
+# a new built version of Hydrant.
+
+import json, requests
+
+from sys import stdin, stdout
+from os import environ, path
+from hmac import digest
+from tempfile import TemporaryDirectory
+from zipfile import ZipFile
+
+LOCKER_DIR = "/afs/sipb.mit.edu/project/hydrant/"
```

Comment:
note that scripts machines don't like `/mit/hydrant`, i think

i haven't actually tested it yet